docs/context-schema/snippets/recipe.bicep

```
@@ -0,0 +1,2 @@
+@description('Radius-provided object containing information about the resource calling the Recipe')
+param context object
```

Let's add a couple example in here for the reader

```suggestion
param context object

var resourceName = context.resource.name
var namespace = context.runtime.kubernetes.namespace
```